src/menu.test.visuals.ts

```diff
@@ -36,6 +37,12 @@ for (const story of stories) {
               element.open = true;
             });
 
+          await page
```

Comment:
This is the `offset` test. Thought it would be good to include the sub-Menu in the screenshot.